airflow/providers/google/cloud/transfers/cassandra_to_gcs.py

```diff
@@ -84,6 +87,10 @@ class CassandraToGCSOperator(BaseOperator):
         Service Account Token Creator IAM role to the directly preceding identity, with first
         account from the list granting this role to the originating account (templated).
     :type impersonation_chain: Union[str, Sequence[str]]
+    :param query_timeout: (Optional) The floating-point timeout used to execute the Cassandra query.
+        If not set, the timeout value will be set in Session.execute() by Cassandra driver.
+        If set to None, there is no timeout.
+    :type query_timeout: Union[float, NotSetType]
```

Comment:
Don't need to mention `NotSetType` here. Documentation is for the user, so you only need to mention the values they need to set.

```suggestion
    :type query_timeout: float | None
```

Reply:
that makes sense. Applied